node/src/accountant/db_access_objects/receivable_dao.rs

```diff
@@ -481,10 +563,10 @@ mod tests {
     #[should_panic(
         expected = "Overflow detected with 340282366920938463463374607431768211455: cannot be converted from u128 to i128"
     )]
-    fn try_multi_insert_payment_handles_error_of_number_sign_check() {
+    fn multi_row_update_from_received_payments_handles_error_of_number_sign_check() {
```

Comment:
I'm worried if these test renames would cause trouble to someone else who have been working on this file in their card. In case someone is, then I'd suggest we keep the old names of these test renames as a comment. If not, feel free to ignore this comment.